### DISCUSSION TITLE
Don't switch target to nearest boardable ship if current target is not disabled

### DISCUSSION THREAD

**Discussion by brendanjones** [endless-sky/endless-sky#7166] [Balance, Mechanics, and Interactions]
This seems to be a recent change, and it's a bug to me. 

Current behaviour:
If I select B (board) when a non-disabled ship is selected, and there is a disabled ship in the system, my selection changes to the disabled ship and boarding action initiates.

Expected behaviour:
If I select B (board) when a non-disabled ship is selected, nothing should happen.

Why? When trying to board a high value ship that's in the middle of an intense battle (e.g. a KIV 349, or an Arfecta), I'm wanting to spam 'B' so that boarding starts asap before the ship gets destroyed. Switching to the nearest boardable ship is entirely unhelpful in this situation. I can imagine the switching to nearest boardable ship is handy is some situations, however, so maybe this could be a preference? Or switching to nearest boardable ship is done with Shift + B?

**Comment by Zern9528** [comment]
I agree, but in the meantime you can spam click on the ship, or try time it. I've gotten fairly good at timing it, but when you miss it's a huge pain and throws everything off.

**Comment by ravenshining** [comment]
I agree that that would be *convenient*, but I don't think we want plundering and capturing to be *more convenient* unless they get rebalanced in some way.

**Reply by brendanjones** [reply]
This I don't agree with - the user's ability to complete an action should be a separate discussion to whether that action is balanced in gameplay or not.

**Comment by mOctave** [comment]
I think that the way it currently gets done should be the default, since new players and players that don't do a whole lot of capturing ships will use boarding that way more often. However, it might be nice to have in settings a way to switch that to the way you describe here.
